apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/template/edit.js

```diff
@@ -156,7 +156,13 @@ const TemplateEdit = compose(
 						{ isSelected && (
 							<Placeholder className="template-block__overlay">
 								<Button href={ editTemplateUrl } onClick={ save } isDefault ref={ navButton }>
-									{ navigateToTemplate ? <Spinner /> : sprintf( __( 'Edit %s' ), templateTitle ) }
+									{ navigateToTemplate ? (
```

Comment:
I am sure there is a more compact way of doing the following without a fragment - but I couldn't find any other combination that would play nice with the ternary - let me know if there is a better way

Reply:
If this gets too big, a helper function is fine to use too